### PR TITLE
Ability to Specify Percentages and Individual H/W in Rendered DIV Dimensions

### PR DESCRIPTION
**Pull request by SendDerek**
See https://github.com/kevinkhill/lavacharts/issues/71 for more details.  I wanted to be able to specfiy a height style, but not the width.  Before this patch, I had to specify both, which ruined the responsive design I was going for.  I also made it possible to specify a percentage '%' instead of just an integer 'px' value.

### CODE REVIEW

**Comment by SendDerek** [comment]
Oh, yuck.  That's a nasty pull request.  Let me try again...

**Comment by kevinkhill** [comment]
Check out what I wrote in issue #71

**Comment by kevinkhill** [comment]
Do you want to clean this up and pull again? I noticed I had left this and the request open. I'm happy to, but idk why there's so many commits in the request (I'm kinda a git novice)

**Comment by SendDerek** [comment]
I'm going to give it a shot now... stay tuned.

**Comment by SendDerek** [comment]
Okay, that was the second attempt.  Not sure what I'm doing wrong, but I'm not really all that excited about spending more time on it.  The enhancement is taken care of with commit https://github.com/SendDerek/lavacharts/commit/aefbef3471942b5c2b9ca4efbc6a2694c68be361 if you'd like to apply the patch yourself to whichever branch you see fit.

**Comment by kevinkhill** [comment]
No worries, when I get a chance, I'll merge your work in manually
